### DISCUSSION TITLE
Dialog on the Big Bang Debate: Dr Rajendra Gupta and Eric Lerner

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#177] [Cosmology Models]
"Dr. Gupta made world-wide headlines in July when he proposed merging the non-expanding tired light hypothesis to explain the Hubble redshift relation with a slower expanding universe model that puts the Big Bang back to 26 billion years ago."

If you have 69 minutes to spare, watch Eric Lerner try to convince Dr. Rajendra Gupta that the Big Bang never happened
[https://youtu.be/HWA273rAKLk](https://youtu.be/HWA273rAKLk)

**Comment by HanDeBruijn** [comment]
Related: ["Reinventing cosmology: New research puts age of universe at 26.7 - not 13.7 - billion years" using tired-light](https://github.com/orgs/a-cosmology-group/discussions/102)

**Reply by bgreenman** [reply]
Debate is a step in the right direction

**Reply by budrap00** [reply]
Dismissal is more appropriate. Debating the BB is like debating Creation Theory. You can't have a scientific debate on metaphysical terms. Both "theories" are belief systems immune to scientific argumentation.

**Reply by RedshiftDrift** [reply]
The problem is that neither Lerner nor Gupta have a model for the "tired-light" redshift.  So they can both fit any set of observations by adjusting their redshift models (Lerner uses a linear model, Gupta uses a logarithmic model).
This means none of their models is falsifiable.

**Comment by mikehelland** [comment]
I'd just like to point out that "debate" isn't necessarily the right word here. Lerner and Gupta had a dialog. It wasn't really competitive and I think we as members of "fringe" should take some guidance there.

We on the fringe often find ourselves at odds each other sometimes more than with the mainstream. If the mainstream is big campfire where everyone is gathered around, we are the ones stoking our little campfires by our selves far away and various distances from the big one, we are often even farther away from each other.

Debate between to fringe positions is basically pointless. No one comes out a winner, and it doesn't attract anyone away from the mainstream.

Lerner's "dialog" approach is better. 

In that spirit, if anyone would like to talk about their ideas on my channel, let's keep it going. Hit me up.

**Reply by bgreenman** [reply]
Dialog is the right term, since we still haven't reached the stage of developing a model for tired light redshift

**Comment by ExpEarth** [comment]
Now Gupta has published a new paper on it, in ApJ no less.  Not that it's any clearer than the first one!

https://phys.org/news/2024-03-universe-dark.html?utm_source=nwletter&utm_medium=email&utm_campaign=weekly-nwletter

**Reply by RedshiftDrift** [reply]
Gupta uses two adjustable parameters:
1) redshift is caused by the expansion of the universe through gravitational drag (some sort of Machian physics), and an exponential functional dependence $\exp[\alpha(t-t_0)]$ is assumed with the adjustable parameter $\alpha$,
2) time-varying physical constants (Dirac's hypothesis), a variation assumed to be following space expansion $C \times a(t)$ where $C = 1$.

With two adjustable parameters he managed to fit two sets of data and publish a paper for each.  I predict that his next papers won't fit new types of data as well (unless he adds another adjustable parameter).
